src/components/SeriesNav/SeriesNavTile.js

```diff
@@ -72,6 +72,13 @@ const SeriesNavTile = ({
 
   return (
     <div
+      /* 
+      Hide PayNote for members by setting the data-hide-if-member attribute. 
+      This class is defined in the MeContext in republik-frontend
+      */
+      {...(PayNote && {
+        'data-hide-if-member': true
```

Comment:
Result will be following HTML
```html
<div data-hide-if-member="true">
```

Same as JSX
```jsx
<div data-hide-if-member>
```

But I saw that you also did the following JSX in the frontend:
```jsx
<div data-hide-if-member=''>
```

Which lead to following *HTML*:
```html
<div data-hide-if-member>
```

I think is true the way to go. But let's make sure we're consistent.